src/buffer/out/UTextAdapter.cpp

```diff
@@ -436,7 +436,7 @@ til::point_span Microsoft::Console::ICU::BufferRangeFromMatch(UText* ut, URegula
     if (utextAccess(ut, nativeIndexEnd, true))
     {
         const auto y = accessCurrentRow(ut);
-        ret.end.x = textBuffer.GetRowByOffset(y).GetTrailingColumnAtCharOffset(ut->chunkOffset);
+        ret.end.x = textBuffer.GetRowByOffset(y).GetLeadingColumnAtCharOffset(ut->chunkOffset);

```

Comment:
@lhecker ?

Reply:
The offsets are exclusive now. This means the chunkOffset now refers to the first character after the matches range. In turn, this means we want the leading half of the wide glyph because we want the first column after the matched range as well.